sql/core/src/main/scala/org/apache/spark/sql/Observation.scala

```diff
@@ -75,23 +79,39 @@ class Observation(name: String) {
   }
 
   /**
-   * Get the observed metrics. This waits for the observed dataset to finish its first action.
-   * Only the result of the first action is available. Subsequent actions do not modify the result.
+   * (Scala-specific) Get the observed metrics. This waits for the observed dataset to finish
+   * its first action. Only the result of the first action is available. Subsequent actions do not
+   * modify the result.
    *
-   * @return the observed metrics as a [[Row]]
+   * @return the observed metrics as a `Map[String, Any]`
    * @throws InterruptedException interrupted while waiting
    */
   @throws[InterruptedException]
-  def get: Row = {
+  def get: Map[String, Any] = {
```

Comment:
sorry last comment. can we use sth like `Map[String, _]`?

Reply:
done